x-pack/plugins/uptime/public/lib/alert_types/monitor_status.tsx

```diff
@@ -44,4 +50,18 @@ export const initMonitorStatusAlertType: AlertTypeInitializer = ({
   },
   defaultActionMessage,
   requiresAppContext: false,
+  format: ({ fields }) => ({
+    reason: fields.reason,
+    link: format({
+      pathname: `/app/uptime/monitor/${btoa(fields['monitor.id']!)}`,
```

Comment:
i think we should abstract out this into a function, something like `getmonitorRouteFromMonitorId` it's also being used in many other places.